flask_jwt_extended/tokens.py

```diff
@@ -38,6 +38,8 @@ def _encode_jwt(
     if isinstance(fresh, timedelta):
         fresh = datetime.timestamp(now + fresh)
 
+    identity_claim_key = identity_claim_keys[0]
```

Comment:
We could encode all of the possible identities into the token, I opted to just use the first listed one as the one that takes precedent since we plan on making the switch over.